code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -65,6 +68,9 @@
 
 
 /datum/game_mode/hivemind/post_setup()
+	if(hosts.len >= 4 && prob(35)) //Create the versus objective here since we want a common target for all the antags
+		assimilation_obj = new /datum/objective/hivemind/assimilate_common
+		assimilation_obj.find_target_by_role(role = ROLE_HIVE, role_type = 1, invert = 1)
```

Comment:
`invert = TRUE`

I know you didn't create this but `role_type` really needs defines (1 does special role and 0 does assigned) but probably for a different PR.